domains/saurabhkirve.json

```
@@ -4,6 +4,8 @@
         "email": "saurabhkirve@gmail.com"
     },
     "record": {
-        "A": ["154.41.233.131"]
-    }
+        "A": ["154.41.233.131"],
+        "CNAME":{ "berzcode.github.io"}
```

You can't have CNAME records with A records. Also, CNAME records need to be a string, not an object.
```suggestion
        "CNAME": "berzcode.github.io"
```